### DISCUSSION TITLE
Show line number on the same line as file name in pretty mode

### DISCUSSION THREAD

**Discussion by timkgh** [BurntSushi/ripgrep#1593] [General] [answered]
Is there a way to have `ripgrep` show the line (and possibly column) number on the same line in pretty mode in the terminal, just like `grep` does? This would allow one to `cmd+click` to open the file in an editor and jump to the matched line.
I know there is `--vimgrep` but I'm looking for pretty output that can be visually processed in a terminal and occasionally `cmd+click` on a file to open it the associated IDE.

**Comment by BurntSushi** [comment, marked as answer]
I think what you want is --no-heading.

Not sure if you'll see this since you deleted the question. I'd appreciate
if you didn't do that. If you found your answer, please post it so that
others can benefit from it too.

Thanks

- Andrew

**Comment by timkgh** [comment]
Thank you. I did not delete the question, it seems GitHub is having issues, it disappears and then it shows up again.

**Comment by BurntSushi** [comment]
Oh! Weird. Sorry about that. I'll check back later and make sure my
answer is posted. :-)

- Andrew
